### ISSUE TITLE
DARK MAGIC: Did not install malware.

### ISSUE THREAD

**Issue by shayneoneill** [basvodde/filezilla#1]
I think theres a problem. I downloaded and built this and it didn't install Malware! What is the meaning of this! How will I compromise my machine and get infected with ask.com without it!

(just kidding. Be good to see a build available in releases however)

**Comment by basvodde** [comment]
Uhm :) This isn't the main Filezilla repository. I think they are still at sourceforge (poor guys). It is a copy of their code with my own refactorings in it... thus won't be released :)

**Comment by ghost** [comment]
I do wish someone would start maintaining Filezilla outside of SourceForge though since SourceForge wraps the installer in malware!

You seem capable of doing this basvodde! How much money do we need to collect to get this to happen?

**Comment by basvodde** [comment]
@waynebloss Uhm, isn't Filezilla maintained by the original maintainers anymore then? I've actually not looked at the project for a long time, this repository originally was meant to show how you can work with legacy code :)

The installer sounds bad though.

**Comment by ghost** [comment]
It is still maintained by the original maintainers, but there is a big controversy going on (at least in the tech news) about projects still being hosted at SourceForge, since they've monetized with malware that nobody wants. The original maintainers of FileZilla have stubbornly refused to move away, even after hundreds of users have requested that they move the project somewhere else. Many suspect that it's because FileZilla is profiting from the people who mistakenly use the installer that has the malware wrapper (since SourceForge pays the project owners an incentive to use the malware installer to begin with).

So, I decided that I'm just not going to use FileZilla at all for now.

**Comment by basvodde** [comment]
Yeah, I'd be uncomfortable hijacking the project, especially since I'm not an active user of Filezilla myself. But I understand the annoyance of the users and do not understand why the Filezilla maintainers haven't moved away from Sourceforge like the rest of the world.
